std/math.d

```diff
@@ -4110,7 +4110,7 @@ real log(real x) @safe pure nothrow @nogc
 ///
 @safe pure nothrow @nogc unittest
 {
-    assert(log(E) == 1);
+    assert(feqrel(log(E), 1) >= real.mant_dig - 1);
```

Comment:
The assert for equality only trips on Android, but not on linux/glibc/AArch64. Since it's a documented test, I can version this only for IEEE Quadruples like the `std.complex` test above and keep this simpler assert for readability if wanted.